core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -847,6 +847,33 @@ class WorkspaceServiceSpec extends AnyFlatSpec with ScalatestRouteTest with Matc
     }
   }
 
+  it should "delete an Azure workspace" in withTestDataServices { services =>
+    val workspaceName = s"rawls-test-workspace-${UUID.randomUUID().toString}"
+    val workspaceRequest = MultiCloudWorkspaceRequest(
```

Comment:
🤔 maybe out of scope for this PR: we should delete the `cloudPlatform` field out of the `MultiCloudWorkspaceRequest` case class, that is an artifact from before we were determining the cloud platform from the billing profile.